publish-hook/src/test/resources/simplelogger.properties

```diff
@@ -0,0 +1,3 @@
+org.slf4j.simpleLogger.log.com.redhat.lightblue=DEBUG
+org.slf4j.simpleLogger.log.com.redhat.lightblue.util.Error=ERROR
+org.slf4j.simpleLogger.defaultLogLevel=ERROR
```

Comment:
Looks like this is producing too many lines for travis. Is it worth removing, or at least upping the error levels?